minetest.conf

```
@@ -170,7 +170,11 @@ spill_removal.lavacooling_method = node
 debug_log_size_max = 0
 # https://namingschemes.com/Star_Wars
 default_password = Endor
-server_description = Pandorabox (default password: Endor)
+server_description = """
+(Use password "Endor" to sign up -- don't use `register` button.)
```

Maybe parens not really needed here?
```suggestion
Use password "Endor" to sign up -- don't use `register` button.
```

---

I felt it was (grammatically speaking) a "side-note" of sorts. No strong opinion either way though :)